sstables/exceptions.hh

```diff
@@ -22,23 +22,33 @@
 
 #pragma once
 
+#include <experimental/source_location>
+
 #include <seastar/core/print.hh>
 
 #include "seastarx.hh"
+#include "log.hh"
```

Comment:
Unneeded include, you can forward declare the logger.

Reply:
removed in v2 and replaced with forward declaration